resources/config.rb

```diff
@@ -0,0 +1,66 @@
+unified_mode true
+
+property :fpm_service, String,
+  default: lazy { php_fpm_service },
+  description: 'Name of the PHP-FPM service to manage.
+Defaults to platform specific names, see libraries/helpers.rb'
+
+property :fpm_ini_control, [true, false],
+  default: lazy { php_fpm_ini_control },
```

Comment:
We don't need to create method helpers for things are just set as a simple true/false. So I'd recommend you remove `php_fpm_ini_control` and just do the following instead:
```suggestion
  default: false,
```